src/App.jsx

```diff
@@ -0,0 +1,46 @@
+import { useState } from 'react'
+import AddTodo from './components/AddTodo'
+import Hints from './components/Hints'
+import Title from './components/Title'
+import TodoInput from './components/TodoInput'
+import TodosList from './components/TodosList'
+import Wrapper from './components/Wrapper'
+
+function App() {
```

Comment:
since you have chosen to use arrow function it can look much better if you also turn this App component to be an arrow function.